config.js

```diff
@@ -9,6 +9,7 @@ var localConfig = extend(true, {
         // [Source, excluding bower_components]: [Destination].
         'document-register-element/build/document-register-element.max.js': config.LIB_DEST_PATH,
         'flipsnap/flipsnap.js': config.LIB_DEST_PATH,
+        'fxpay/dist/fxpay.min.js': config.LIB_DEST_PATH,
```

Comment:
right now we can't do that because all the fxpay modules aren't requireJS compliant. However, I want to make them so as it will unlock many possibilities.